packages/json-schema/scripts/convert-schemas.mjs

```diff
@@ -0,0 +1,15 @@
+import fs from 'node:fs';
+import { join } from 'node:path';
+import { zodToJsonSchema } from 'zod-to-json-schema';
+import { VERSIONED_CONFIG_SCHEMAS } from '@vitessce/schemas/dist/index.mjs';
```

Comment:
It is required for Node to recognize that file as containing ESM code

Reply:
I think there is an underlying issue with our `package.json` not being configured correctly with `type: module` but probably should tackle in a different PR

Reply:
Ah I see now why. It is because here we want to use Node to import from `@vitessce/schemas` but we are using the development version of the package. Once the package has been bundled for production, then the PNPM [publishConfig](https://pnpm.io/package_json#publishconfig) will take care of this. I have added a comment about this.

I have also cleaned up the `package.json` files to correctly specify `type: module` as well, but it turns out to be unrelated to this